app/controllers/hooks_controller.rb

```diff
@@ -13,6 +13,18 @@ def github
     render json: nil, status: :ok
   end
 
+  def package
+    if params[:payload]
+      payload = JSON.parse(params[:payload])
```

Comment:
Had this for the `github` endpoint, will remove since we always control the input for the `package` endpoint